docs/Migration-from-the-AuthComponent.md

```diff
@@ -33,9 +33,17 @@ The AuthenticationMiddleware will handle checking and setting the identity based
 ```php
 function login()
 {
+    $result = $this->Authentication->getResult();
+
     // regardless of POST or GET, redirect if user is logged in
-    if ($this->Authentication->getResult()->isValid()) {
-        return $this->redirect('/');
+    if ($result->isValid()) {
+        $redirect = $this->request->getQuery('redirect', ['controller' => 'Pages', 'action' => 'display', 'home']);
```

Comment:
Perhaps use string URL instead of array? We are trying to keep this plugin decoupled from Cake.

Reply:
But meh, the whole example is Cake specific.

Reply:
I would rather see more URL arrays then just strings, especially in the CakePHP scope.

Reply:
Or named routes :bike: